src/main/java/graphql/execution/instrumentation/parameters/InstrumentationParsedNormalizedOperation.java

```diff
@@ -0,0 +1,17 @@
+package graphql.execution.instrumentation.parameters;
+
+import graphql.ExecutionInput;
+import graphql.PublicApi;
+import graphql.execution.instrumentation.Instrumentation;
+import graphql.schema.GraphQLSchema;
+
+/**
+ * Parameters sent to {@link Instrumentation} methods
+ */
+@SuppressWarnings("TypeParameterUnusedInFormals")
+@PublicApi
+public class InstrumentationParsedNormalizedOperation extends InstrumentationExecutionParameters {
+    public InstrumentationParsedNormalizedOperation(ExecutionInput executionInput, GraphQLSchema schema) {
+        super(executionInput, schema);
+    }
+}
```

Comment:
All the classes are named InstrumentationXXXParameters

Please rename to InstrumentationParsedNormalizedOperationParameters


InstrumentationCreateStateParameters
InstrumentationExecuteOperationParameters
InstrumentationExecutionParameters
InstrumentationExecutionStrategyParameters
InstrumentationFieldCompleteParameters
InstrumentationFieldFetchParameters
InstrumentationFieldParameters
InstrumentationValidationParameters